### PR TITLE
[4.0]Remove switcher from admin menu edit

### PR DESCRIPTION
**Pull request by chmst**
Related to #22866 .

### Summary of Changes
Use the selection of the context when opening the ed view for a new menu.
Then the switcher is not needed any more. 
If the user is in administrator context he/she can choose a preset. If in site context, the preset is not rendered. 
menu-admin

### Testing Instructions
Go to manage menu and edit a menu or make a new one. 

### Expected result
If you are in context site, you cannot choose a preset, if you are in context administrator you can choose.

### Actual result
See #22866.
Additional remark - when tabbing through the screen, it is not possible to open the selection for presets

### Documentation Changes Required
yes

### CODE REVIEW

File: administrator/components/com_menus/Model/MenuModel.php [hunk 1]
```diff
@@ -104,6 +104,10 @@ protected function populateState()
 		// Load the parameters.
 		$params = ComponentHelper::getParams('com_menus');
 		$this->setState('params', $params);
+
```
Comment (Quy) [inline administrator/components/com_menus/Model/MenuModel.php]:
```suggestion

```

File: administrator/components/com_menus/Model/MenuModel.php [hunk 2]
```diff
@@ -177,6 +181,11 @@ protected function loadFormData()
 		if (empty($data))
 		{
 			$data = $this->getItem();
+
```
Comment (Quy) [inline administrator/components/com_menus/Model/MenuModel.php]:
```suggestion

```

File: administrator/components/com_menus/forms/menu.xml [hunk 3]
```diff
@@ -42,21 +42,13 @@
...
-		</field>
+			type="hidden"
+		/>
 		<field
 			name="preset"
 			type="menuPreset"
 			label="COM_MENUS_FIELD_PRESET_LABEL"
 			description="COM_MENUS_FIELD_PRESET_DESC"
-			showon="client_id:1"
 		>
```
Comment (Quy) [inline administrator/components/com_menus/forms/menu.xml]:
```suggestion
			>
```

**Review by brianteeman — APPROVED** [review]
approve language change

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fd000b1b65d386fd52403477e215fcaae54f8482<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22903">issues.joomla.org/tracker/joomla-cms/22903</a>.</sub>

**Pull request merged**